gen/dcos-config.yaml

```diff
@@ -1926,6 +1926,14 @@ package:
               {
                   "Command": ["/opt/mesosphere/bin/dcos-shell", "dcos-etcdctl", "diagnostic"],
                   "Role": ["master"]
+              },
+              {
+                  "Command": ["/bin/sh", "-c", "source /run/dcos/etc/etcd/zk-creds; echo -e \"addauth digest ${DATASTORE_ZK_USER}:${DATASTORE_ZK_SECRET}\\nget /etcd/nodes\" | /opt/mesosphere/bin/dcos-shell zkCli.sh 2>&1"],
```

Comment:
are these variables accessible from this file?

Reply:
Yes, I also verified it works by invoking the same cmdline from the masters.